modules/ROOT/pages/migration/upgrading_4.0.0_5.0.0.adoc

```diff
@@ -105,3 +106,21 @@ NOTE: For new deployments starting with ocis 5.0.0, these values are autogenerat
 
 WARNING: If no values are provided during an upgrade, ocis will not start and log a fatal error.
 --
+
+=== Deprecations
+
+* Service Registies
++
+--
+We deprecated some service registries. If your `MICRO_REGISTRY` config is set to one of these values `mdns, nats, kubernetes, etcd, consul` please use `nats-js-kv` in the future (`memory` is only intended for testing environments).
+--
+* Micro caches and stores
++
+--
+We deprecated some micro caches and stores. If one of your `*_CACHE_STORE` variables is using one of there values `redis-sentinel, redis, etcd, nats, ocmem` please use `nats-js-kv` in the future (`memory` is only intended for testing environments).
+--
+* Resharing
++
+--
+We have disabled the resharing feature by default. It will be removed from the product. Existing reshares will still be visible to the original resource owner. Creation of new reshares will not be possible. Please make sure that `OCIS_ENABLE_RESHARING` is not set to `true` in your deployments.
```

Comment:
@ScharfViktor Is testing that currently. Please change if my "statement" is not true 😄